rero_ils/modules/operation_logs/cli.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+#
+# RERO ILS
+# Copyright (C) 2021 RERO
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, version 3 of the License.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""Click command-line interface for operation_log record management."""
+
+from __future__ import absolute_import, print_function
+
+import json
+
+import click
+from flask.cli import with_appcontext
+
+from rero_ils.modules.operation_logs.api import OperationLog
+from rero_ils.modules.operation_logs.models import OperationLogOperation
+from rero_ils.modules.utils import extracted_data_from_ref
+
+
+@click.command('migrate_virtua_operation_logs')
+@click.option('-f', 'input_file', help='Logs input file')
+@with_appcontext
+def migrate_virtua_operation_logs(input_file):
+    """Migrate Virtua operation logs to REROILS."""
+    with open(input_file, 'r', encoding='utf-8') as request_file:
+        oplgs = json.load(request_file)
+        for oplg in oplgs:
+            operation = oplg.get('operation')
+            if operation == OperationLogOperation.CREATE:
+                record_pid = extracted_data_from_ref(
+                    oplg.get('record').get('$ref'), data='pid')
+                resource = extracted_data_from_ref(
+                    oplg.get('record').get('$ref'), data='resource')
+                create_rec = \
+                    OperationLog.get_create_operation_log_by_resource_pid(
+                        resource, record_pid)
+                if create_rec:
+                    create_rec.update(oplg, dbcommit=True, reindex=True)
```

Comment:
We have `OperationLog. update ` but we in `operationLogOperation. CREATE ` ?

Reply:
It is normal, I added a code comment to explain this